node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,8 +137,11 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, and types that the browser has plugins to support
+            var supportedTypes = _.union(_.pluck(navigator.mimeTypes, 'type'), ['text/html', 'text/plain']);
+            var supported = _.contains(supportedTypes, widgetData.previews.targetType.split(';')[0]);
```

Comment:
We'll need to be able to deal with a `null` widgetData.previews.targetType value for backwards-compatibility reasons